### PR TITLE
[fix] resize_, resize_as_ : version bump in ADInplaceOrView

### PR DESCRIPTION
**Pull request by kshitij12345**
Ref: https://github.com/pytorch/pytorch/pull/96403#discussion_r1132553277

### CODE REVIEW

**Review by albanD — COMMENTED** [review]
Awesome!
Could you update the test at https://github.com/pytorch/pytorch/blob/3cdf18cb4f1fc46ef301ec615821ffb5a1a5ed66/test/test_autograd.py#L9996 to also run the inplace with all the manual functions? (copy_, resisze_ and resize_as_)  To make sure they all work properly.

File: test/test_autograd.py [hunk 1]
```diff
@@ -9995,6 +9995,11 @@ def run_test(fn):
         run_test(lambda x: x.add_(2))
         run_test(lambda x: x.transpose_(0, 1))
 
+        # inplace ops with manual kernel for ADOrInplace key in VariableTypeManual.cpp
```
Comment (albanD) [inline test/test_autograd.py:9998]: nit: ADInplaceOrView

**Review by albanD — APPROVED** [review]
Thanks!

**Comment by kshitij12345** [comment]
@albanD is it ok to force merge? (Since previous commit was green and latest commit is just typo fix)

**Pull request closed without merging**